pkgs/development/python-modules/tinygrad/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "tinygrad";
+  version = "0.7.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "tinygrad";
+    repo = "tinygrad";
+    rev = "v${version}";
+    hash = "sha256-lXpMqG8yNYkYZ0LZh5ERYCViELTauwiijDRAmQZIfQI=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+  ];
+
+  pythonImportsCheck = [ "tinygrad" ];
+
+  meta = with lib; {
+    description = "You like pytorch? You like micrograd? You love tinygrad";
+    homepage = "https://github.com/tinygrad/tinygrad";
```

```suggestion
    homepage = "https://github.com/tinygrad/tinygrad";
    changelog = "https://github.com/tinygrad/tinygrad/releases/tag/${src.rev}";
```
Adding the changelog is useful for future updates.